llvm/test/CodeGen/Hexagon/swp-ws-pragma-initiation-interval-fail.mir

```diff
@@ -0,0 +1,85 @@
+# RUN: llc  --march=hexagon %s -run-pass=pipeliner -debug-only=pipeliner \
+# RUN: -window-sched=force -filetype=null -verify-machineinstrs 2>&1 \
+# RUN: | FileCheck %s
+# REQUIRES: asserts
+
+# Test that checks no window scheduler is performed if the II set by pragma was
+# enabled
+
+# CHECK-NOT: Start analyzing II
+# CHECK-NOT: Start scheduling Phis
+# CHECK-NOT: Current window Offset is {{[0-9]+}} and II is {{[0-9]+}}
```

Comment:
We have added new debug information to report that the window scheduler is not supported when the initiation interval is set by pragma. Since window scheduling is not operated, this debug information is the only way to determine the status of the window scheduler.